.github/workflows/build.yml

```diff
@@ -148,6 +148,15 @@ jobs:
           make build-jupyterlab
       - name: Cypress
         run: make test-integration
+      - name: Collect logs
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          path: |
+            ${{ github.workspace }}/build/cypress-tests/*.log
+            ${{ github.workspace }}/build/cypress-tests/screenshots//**/*
+            ${{ github.workspace }}/build/cypress-tests/videos//**/*
```

Comment:
Out of curiosity what does the double slash do? “//“

Reply:
Not sure! I didn't write this code originally, just copied it back in

Reply:
I'm pretty sure the double-slash is not necessary.  Here's the path syntax cheat sheet: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-file-paths